internal/pkg/stream/cfnstackset.go

```diff
@@ -118,24 +114,22 @@ func (s *StackSetStreamer) Subscribe() <-chan StackSetOpEvent {
 }
 
 // Fetch retrieves the latest stack set operation and buffers it.
```

Comment:
Can we update the comment in the other `Fetch` methods too to explain the purpose of the boolean return variable 🥺